src/components/Personkort.tsx

```diff
@@ -35,3 +37,34 @@ export const Personkort = (props: { person: Person }) => {
         </div>
     );
 };
+
+export const PersonkortEPS = (props: { eps: Nullable<SatsBehandlingsinfo> }) => {
+    if (!props.eps || !props.eps.navn || !props.eps.fnr) {
+        return <></>;
+    }
+
+    return (
+        <div className={styles.personkortContainer}>
+            <div>
+                <span className={styles.personkortSVG}>
+                    {props.eps.kjønn === Kjønn.Kvinne ? (
+                        <KjønnKvinne />
+                    ) : props.eps.kjønn === Kjønn.Mann ? (
+                        <KjønnMann />
+                    ) : (
+                        <KjønnUkjent />
+                    )}
+                </span>
+            </div>
+            <div>
+                <p>{`${props.eps.navn.fornavn} ${props.eps.navn.mellomnavn} ${props.eps.navn.etternavn}`}</p>
+                <div>
+                    <span>{`${props.eps.fnr.substring(0, 6)} `}</span>
+                    <span>{`${props.eps.fnr.substring(6, 11)}, `}</span>
+                    <span>{props.eps.alder} år</span>
+                </div>
```

Comment:
Er det meningen at vi skal vise dette på en helt annen måte enn vi gjør i `Personkort`? Både mtp. fødselsnummer og navn.

Reply:
Navn er vel like? Vi bruker vel bare en funksjon som gjør det for oss på det andre personkortet. 

Men ja, de skulle være annerledes i skissene på sats.